### ISSUE TITLE
[RFE] export function for ACC convert (vscode-f5-chariot)

### ISSUE THREAD

**Issue by DumpySquare** [f5devcentral/f5-as3-config-converter#26]
### request
As discussed, please export a function in the base node project to allow for direct api access in vscode-f5-chariot.

Here is how I am re-wrapping the ACC functionality for the extension:  https://github.com/f5devcentral/vscode-f5-chariot/blob/main/src/accWrapper.ts

Right now, the function just takes in the raw tmos config (no archive extraction), sets some ACC parsing options, and return the declaration output with metadata.  

If there are other things that we need to added to the output for your troubleshooting, please add them as part of the return.  The extension can log the metadata as needed.

This will allow me to import your latest ACC function to the vscode-f5-chariot project and quickly release updates.  Thanks.

### statically setting config conversion options
```ts
    const config = {
        ucs: false,
        container: false,
        disableAnalytics: false,
        recognized: true,
        recognizedObjects: true,
        supported: true,
        supportedObjects: true,
        unsupported: true,
        unsupportedObjects: true,
        summary: true,
        debug: true,
        showExtended: true
    };
```

### function output
```ts
    return {
        declaration,
        metaData: {
            recognized: as3Json,
            supported: supportedJson,
            unSupported: unsupportedObj,
            declarationInfo,
        }
    };
```

**Comment by mdditt2000** [comment]
Created Jira CHARON-314 for PM tracking.

**Comment by mdditt2000** [comment]
@DumpySquare this was picked up by @v-zhuravlev-f5 Changing to ACC 1.13 -- Woot Woot!!

**Comment by mdditt2000** [comment]
Merge branch '314-chariot-function' into 'develop' -- Resolved in ACC 1.13
